docs/DeveloperGuide.adoc

```diff
@@ -18,6 +18,7 @@ endif::[]
 By: `Team F10-2`      Since: `Sep 2018`      Licence: `MIT`
 
 == Setting up
+This section describes the necessary dependencies required to install the application, together with the steps required to install.
```

Comment:
We are not installing the application in the Dev Guide, rather we are setting up the tools necessary to work on the project's codebase.